driver/driver_options_test.go

```diff
@@ -94,16 +97,85 @@ func TestDriverOptions_writeTimeout(t *testing.T) {
 	srv := CreateMockServer(t)
 	defer srv.Stop()
 
+	// use a writeTimeout that will fail parsing by ParseDuration resulting
+	// in a conn open error. The Open() won't fail until the ExecContext()
+	// call though, because that's when golang database/sql package will open
+	// the actual connection.
 	conn, err := sql.Open("mysql", "root@127.0.0.1:3307/test?writeTimeout=10")
 	require.NoError(t, err)
+	require.NotNil(t, conn)
 
-	result, err := conn.ExecContext(context.TODO(), "insert into slow(a,b) values(1,2);")
+	// here we should fail because of the missing time unit in the duration.
+	result, err := conn.ExecContext(context.TODO(), "select 1;")
+	require.Contains(t, err.Error(), "missing unit in duration")
+	require.Error(t, err)
 	require.Nil(t, result)
+
+	// use an almost zero (1ns) writeTimeout to ensure the insert statement
+	// can't write before the timeout. Just want to make sure ExecContext()
+	// will throw an error.
+	conn, err = sql.Open("mysql", "root@127.0.0.1:3307/test?writeTimeout=1ns")
+	require.NoError(t, err)
+
+	// ExecContext() should fail due to the write timeout of 1ns
+	result, err = conn.ExecContext(context.TODO(), "insert into slow(a,b) values(1,2);")
 	require.Error(t, err)
+	require.Contains(t, err.Error(), "i/o timeout")
+	require.Nil(t, result)
 
 	conn.Close()
 }
 
+func TestDriverOptions_namedValueChecker(t *testing.T) {
+	AddNamedValueChecker(func(nv *sqlDriver.NamedValue) error {
+		rv := reflect.ValueOf(nv.Value)
```

Comment:
I also checked it can be written as

```
		_, ok := nv.Value.(uint64)
		if !ok {
			return sqlDriver.ErrSkip
		}

		return nil
```

Maybe it's faster to use golang's built-in type cast rather than reflection in your application.

Reply:
Oh yea thanks. This is much better. 

Unrelated question, how often do you, or what is the criteria to, create a tag for a new version?

Reply:
As a user, our project always uses "master" branch 😂 So generally when other contributors ask for a release I ping @atercattus to release it.